ONSdigital/eq-survey-runner
# Update travis nod version and declare dependencies

**_What**_

1) Switches the node version used by travis to match that on local development machines.
2) Explicitly defines some dependencies that were getting lost in dependency hell

**_How to test**_

1) Check out
2) `rm -rf node_modules`
3) `npm install`
4) `npm run compile`
5) Verify no errors

**_Who can test**_

https://www.youtube.com/watch?v=KBKXu3Kg4yg
